tool/amdllpc.cpp

```diff
@@ -1417,17 +1417,6 @@ void FindAllMatchFiles(
     FindClose(pSearchHandle);
     return;
 }
-#else
```

Comment:
How to hit this case? actually, I implemented the real implmentation on linux when I add this function.
But when I run the test on linux, I found linux shell expand all * and ?, I replace it with an assert.

Reply:
1. You can hit it by saying amdllpc "*.pipe" (i.e. quote the * so it does not get expanded).
2. If you do not quote the * but the shell finds no matching file for the glob *.pipe, the default setting of the shell is to pass the *.pipe verbatim through to the command, so the command will say "*.pipe: file not found".

Reply:
In (2), that was supposed to say:
the default setting of the shell is to pass the `*.pipe` verbatim through to the command

Pesky markdown mangled my comment. :-)